tests/baselines/reference/binaryArithmatic2.errors.txt

```diff
@@ -1,7 +1,7 @@
-tests/cases/compiler/binaryArithmatic2.ts(1,13): error TS2532: Object is possibly 'undefined'.
+tests/cases/compiler/binaryArithmatic2.ts(1,13): error TS18047: 'undefined' is possibly 'undefined'.

```

Comment:
This PR isn't the place for this fix, but this error is both redundant and too conservative ("possibly"?).